target/linux/ath79/image/generic.mk

```diff
@@ -328,3 +328,16 @@ define Device/wd_mynet-wifi-rangeextender
   SUPPORTED_DEVICES += mynet-rext
 endef
 TARGET_DEVICES += wd_mynet-wifi-rangeextender
+
+define Device/rosinson_wr818
+  ATH_SOC := qca9563
+  DEVICE_TITLE := ROSINSON WR818
+  KERNEL := kernel-bin | append-dtb | lzma | uImage lzma
+  KERNEL_INITRAMFS := kernel-bin | append-dtb | lzma | uImage lzma
+  IMAGE_SIZE := 15936k
+  IMAGES := sysupgrade.bin
+  IMAGE/default := append-kernel | append-rootfs | pad-rootfs
+  IMAGE/sysupgrade.bin := $$(IMAGE/default) | append-metadata | check-size $$$$(IMAGE_SIZE)
+  DEVICE_PACKAGES := kmod-leds-reset kmod-usb-core kmod-usb2 kmod-usb-storage
```

Comment:
You are using the usual gpio leds, the kmod-leds-reset isn't required.

We usually don't include the `kmod-usb-storage` in the image. It is up to the user to download and install this package. Furthermore, without a filesystem kmod, it doesn't make much sense to include the storage module.